lib/shared/src/models/index.ts

```diff
@@ -24,8 +24,7 @@ export class ModelProvider {
         // The usage of the model, e.g. chat or edit.
         public readonly usage: ModelUsage[],
         // The maximum number of tokens that can be processed by the model in a single request.
-        // NOTE: A token is equivalent to 4 characters/bytes.
-        public readonly maxToken: number = DEFAULT_CHAT_MODEL_TOKEN_LIMIT
+        public readonly maxToken: number = CHAT_TOKEN_BUDGET
```

Comment:
```suggestion
        public readonly maxRequestTokens: number = CHAT_TOKEN_BUDGET
```

or `maxTokens` or something like that. `maxToken` sounds like "the biggest value a token could have."

I like `maxRequestTokens` because some models have different input and output token limits. If we have to handle them specially later, the purpose of `maxRequestTokens` is clear but `maxTokens` could be ambiguous (is it input, output, requests, something else?)